health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -0,0 +1,440 @@
+                                // Check if the index is less than defData length
+                                const isLessThanDefData = index < defData.length;
+            
+                                if (isLessThanDefData) {
+                                    if (hierItem?.boundaryType === defData[index]?.boundaryType) {
+                                        // console.log("hehe"); // This will log when the condition is met
+                                        return (
+                                            <div>
+                                                <div style={{ fontWeight: 400 }} key={index}>
+                                                    {hierItem?.boundaryType}
+                                                </div>
+                                                <div  onClick={() => {}}>
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)